marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/flow.sjs

```diff
@@ -249,6 +249,9 @@ class Flow {
         }
       }
     }
+    if (prov.granularityLevel() !== prov.OFF_LEVEL) {
```

Comment:
Is the reason for this that mlRunIngest.sjs sets "noBatchWrite" to true?

Reply:
Yeah and because of that the prov docs aren't getting written